packages/orchestration/src/exos/chainAccountKit.js

```diff
@@ -134,7 +138,9 @@ export const prepareChainAccountKit = zone =>
         },
         async deposit(payment) {
           console.log('deposit got', payment);
-          throw new Error('not yet implemented');
+          console.error(
+            'FIXME deposit noop until https://github.com/Agoric/agoric-sdk/issues/9193',
+          );
```

Comment:
I don't think we'll have `deposit` or `getPurse` with 9193. The plan is to demonstrate  how to use a LocalChainAccount to deposit (and transfer) ERTP assets to a ChainAccount.